src/content/docs/codestream/codestream-integrations/msteams-integration.mdx

```diff
@@ -3,17 +3,17 @@ title: Microsoft Teams and CodeStream
 description: "How to connect CodeStream to your Microsoft Teams channel."
 ---
 
-import mstcsAppPage from 'images/MSTCSAppPage.png'
+import codestreamMsteamsCodestreamAppPage from 'images/codestream_screenshot-crop_msteams-codestream-app-page.png'
 
-import mstTokenMsg from 'images/MSTTokenMsg.png'
+import codestreamMsteamsTokenExample from 'images/MSTTokenMsg.png'
```

Comment:
I don't know if this is a huge deal @paperclypse but this doesn't look like it follows the correct naming convention?